theme/static/js/aggregation_files_to_sync.js

```diff
@@ -0,0 +1,69 @@
+let aggrFilesToSyncDiv = $("#aggr-files-to-sync");
```

Comment:
might want to wrap in `$(document).ready(function () {});`

Reply:
Done